features/manage-position/YourPosition.tsx

```diff
@@ -121,37 +121,40 @@ const YourPosition = () => {
           {`${_collateral} ${_collSymbol}`}
         </Status>
         <Status>
-          <Label>Tokens outstanding: </Label>
+          <Label>Tokens created: </Label>
           {`${_tokens} ${_tokenSymbol}`}
         </Status>
         <Status>
           <Label>
-            Estimated identifier price (
+            Identifier price(
             <Link href={pricefeedUrl} target="_blank" rel="noopener noreferrer">
-              Coinbase Pro
+              Coinbase
             </Link>
             ):{" "}
           </Label>
-          {_latestPrice}
+          {Number(_latestPrice).toFixed(4)}
         </Status>
         <Status>
-          <Label>(CR) Collateralization ratio:</Label>
-          {` ${pricedCR} (${_collSymbol} / ${_tokenSymbol})`}
+          <Label>Collateralization ratio(CR):</Label>
+          {` ${pricedCR}`}
         </Status>
         <Status>
-          <Label>(GCR) Global collateralization ratio:</Label>
-          {` ${pricedGCR} (${_collSymbol} / ${_tokenSymbol})`}
+          <Label>Global collateralization ratio(GCR):</Label>
+          {` ${pricedGCR}`}
         </Status>
         <Status>
           <Label>Liquidation Requirement:</Label>
           {` ${collReqFromWei}`}
         </Status>
         <Status>
-          <Label>Liquidation Price:</Label>
-          {` ${liquidationPrice} (${priceIdUtf8})`}
+          <Label>Liquidation price:</Label>
+          {` ${liquidationPrice}(${priceIdUtf8})`}
```

Comment:
Why do you prefer to not have spaces between values and their symbols?

Reply:
mainly wanted the width to be narrow.

Reply:
main was trying to save as much width as posible